### ISSUE TITLE
No usage section in the readme

### ISSUE THREAD

**Issue by ricklancee** [GoogleChrome/simplehttp2server#23]
It's quite obvious for most developers how to use simplehttp2server, just run the `simplehttp2server` command after installing it in the directory you want to serve. However when first reading the docs i was instinctively looking (skimming 😄) for a "usage section" and thinking "how do i run this?" 

Being a developer i just ran the command in terminal and of course that worked. However especially for developers that are quite new to the terminal i would reckon they would be quite puzzled since there's not really a usage section in the readme. There's only a section about the config flag that implies you can run `simplehttp2server -config`. 

**tl;dr:** Maybe add a section between Docker and Config (example below) that explains usage of simplehttp2server more explicitly. Thoughts on this?

## Usage

Run the `simplehttp2server` command from the directory you want to serve to https://localhost:5000 (default)
```
simplehttp2server [options]
options: 
  -config     string     Config file
  -cors       string     Set allowed origins (default "*")
  -listen     string     Port to listen on (default ":5000")
```

**Comment by surma** [comment]
Huh... what an _odd_ concept. A “usage” section? Unheard of!



... okay I screwed up. Will add
